ui/src/components/bounding-box-tool/resize-anchor.component.tsx

```diff
@@ -0,0 +1,94 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+import { CSSProperties, ReactNode } from 'react';
+
+import { Anchor as InternalAnchor } from './anchor.component';
+
+export const ANCHOR_SIZE = 8;
+
+enum ResizeAnchorType {
+    SQUARE,
+    CIRCLE,
+    CUSTOM,
+}
+
+interface ResizeAnchorProps {
+    zoom: number;
+    x: number;
+    y: number;
+    moveAnchorTo: (x: number, y: number) => void;
+    cursor?: CSSProperties['cursor'];
+    label: string;
+    onStart?: () => void;
+    onComplete: () => void;
+    type?: ResizeAnchorType;
+    fill?: string;
+    angle?: number;
```

Comment:
Since we don’t support rotated bounding boxes, I don’t think we need this

Reply:
Updated